Directory.Build.targets

```diff
@@ -17,6 +17,9 @@
     <PackageReference Update="Microsoft.Extensions.Logging" Version="2.0.0" />
     <PackageReference Update="Microsoft.Extensions.Logging.Debug" Version="2.0.0" />
     <PackageReference Update="Microsoft.Extensions.Configuration" Version="2.0.0" />
+    <PackageReference Update="Microsoft.Extensions.Configuration.Binder" Version="2.0.0" />
```

Comment:
What do the new dependencies enable from a consumers perspective?

Reply:
lights up some of the options behavior, and the new configure methods, but strictly isn't needed.  And I'm not sure I want to bring these changes in with a patch.

Reply:
So these are in here, however the example only exists in the unit tests.  It would be on the consumer to pull them in until the next version.

Reply:
Without them are consumers of the O# framework library still able to bring them in themselves and reap the benefits if they write the code themselves? Wondering if it adds a lot of value to have them part of the base framework is all

Reply:
I think it has value, however I want to minimize the impact so this can be released as a patch fairly soon.